en/docs/release-notes.md

```diff
@@ -1,5 +1,24 @@
 # Release Notes
 
+## Jun 23, 2022
+### API updates
+1. Added query parameter 'scope' for GET /report/flow and GET /report/requests.
+2. Updated APIs of service quota to meet the requirements of trial customers.
+3. Added a scheduler to check trial deadline and traffic usage of trial customers, and send out notifications.
+4. Added a scheduler to disable transport service when meets trial deadline or traffic limit.
+5. Reorganized the acceleration regions of existing transports.
+6. Made API query parameters case insensitive for easy use.
+
+### Portal updates
+1. Added a notification module.
```

Comment:
This is OK though it would be better if we wrote it in a way that explained the benefit to the customer.

Reply:
Replace the sentence with:
"Added notifications module for pushing news about service updates and marketing promotions."